lmcache/v1/storage_backend/local_cpu_backend.py

```diff
@@ -350,8 +350,16 @@ def initialize_allocator(
                 ]
             )
             paged_mem_allocator = PagedCpuGpuMemoryAllocator()
+            chunk_size_bytes = (
```

Comment:
You can abstract this to a method and place to `utils.py` to reduce redundant code, AFAIK, there is a same calculation in the `RemoteBackend`.